README.md

```diff
@@ -154,42 +160,21 @@ Example commands:
 
 Install this project with:
 
+cd my-project
+  npm install
 <!--
 Example command:
 
 ```sh
   cd my-project
-  gem install
-```
---->
-
-### Usage
-
-To run the project, execute the following command:
-
-<!--
-Example command:
-
-```sh
-  rails server
-```
---->
-
-### Run tests
-
-To run tests, run the following command:
-
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
+  npm install
 ```
 --->
 
 ### Deployment
 
 You can deploy this project using:
+-**Github Pages**
```

Comment:
- [ ] Kindly you should add your live demo link in the Live Demo section of your README file. To do that you should use GitHub Pages to deploy the web pages and make sure that all pages, links, and images work as expected. It would make your portfolio looks more professional :+1: